maintainer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Have you tested this bash script? Does it work?

Reply:
thinking about removing it entirely